Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -112,41 +64,74 @@
                   "durationMs": 7776000000
                 }
               ]
-            }
+            },
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "ff00e059-7ba9-4dfa-a926-8288cf30fc05",
+            "version": "KqlParameterItem/1.0",
+            "name": "UsersList",
+            "label": "Show occurrences for",
+            "type": 2,
+            "description": "A cohort filter can be set at edit mode that is a collection of users, sessions, or operations that are matched by a set of filters specified in the definition of the cohort. When a cohort is used in this report, the result will be the intersection of users, sessions, or operations matched by the cohort and users, sessions, or operations matched by the rest of the query in this report.",
+            "isRequired": true,
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "jsonData": "[{\"value\":\"user_Id\",\"label\":\"All Users\",\"group\":\"\",\"selected\":\"true\"}]",
+            "resourceType": "microsoft.insights/components"
+          },
+          {
+            "id": "0afb0089-98e9-479c-b12e-a982870ce6be",
+            "version": "KqlParameterItem/1.0",
+            "name": "Whoused",
+            "label": "Who used",
+            "type": 2,
+            "description": "Choose which custom events and page views the users should have sent for this report.",
+            "isRequired": true,
+            "query": "datatable(value:string, text:string, selected: string)[\r\n'pageViews,customEvents,requests', 'Any Custom Events, Request or Page View', '',\r\n'pageViews,customEvents', 'Any Custom Events or Page View', 'true',\r\n'customEvents','Any Custom Event', '',\r\n'pageViews','Any Page View', '',\r\n'requests','Any Request', ''\r\n]",
```

Comment:
This could just be json right? No need to be a query

Reply:
will update at next